i18n/es_ES.csv

```diff
@@ -160,12 +160,12 @@
 "Magento Enterprise Edition version","Versión Edición de empresa de Magento"
 "Quick reference guide","Guía de referencia rápida"
 "Log in to the ","Iniciar sesión en"
-"Buckaroo Payment Plaza","Payment Plaza de Buckaroo"
+"Buckaroo Buckaroo Plaza","Buckaroo Plaza de Buckaroo"
```

Comment:
this one too :), sry for the comments, maybe they will be helpful to identify these typos